internal/provider/resource_managed_identity_alias.go

```
@@ -124,12 +134,27 @@ func (t *managedIdentityAliasResource) Create(ctx context.Context,
 		return
 	}
 
+	var sourceIdentityID, sourceIdentityPath *string
+
+	if managedIdentityAlias.AliasSourceID.ValueString() != "" {
```

We'll need to return an error if both are specified.